### PR TITLE
Changes to skilldirectoryUI

### LINKED ISSUE #109: Redesign the Current Skill Display
Will provide a mock up with Skill Icon and all elements

https://wireframe.cc/fDtf9b

### LINKED ISSUE #128: Make an Icon component
We could easily make an `<Icon />` component to abstract the implementation away (and to get rid of the conditional rendering in `<Skills />`

### PR DESCRIPTION
**Pull request by smokinGun**
Reformatted the user interface: Added styling to the review panels. Can now change Icon by double clicking on the icon itself (with default image if no image is selected e.g. a new skill) and abstracted icon into a new component. Reformatted AddReview and AddLink buttons.

Fixes #109 and Fixes #128.
screen shot 2017-02-14 at 12 01 26 pm

### CODE REVIEW

File: src/AppDemo.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        <Route path="home" component={Home} />
+        <Route path="skills(/:id)" component={SkillsUIDemo} />
+        <Route path="team(/:id)" component={Team} />
+        <Route path="login" component={Login} />
+      </Route>
+    </Router>
+  );
+}
+
+export default App;
```
Comment (dane-johnson) [inline src/AppDemo.jsx]: What is this file for?

File: src/ReviewPanel.jsx [hunk 2]
```diff
@@ -1,32 +1,29 @@
...
 
 const ReviewPanel = ({review}) => {
   if(!review) return <div />;
+
+  const body=review.body;
   const time = review.timestamp;
   const good = review.positive;
   const reviewer = review.team_member_name;
   const header = reviewer + " on " + new Date(time).toDateString();
-  let flags = null;
```
Comment (dane-johnson) [inline src/ReviewPanel.jsx:10]: Maybe we should have someplace for badges, likes, and other doohickeys here.

File: src/SkillUIDemo.jsx [hunk 3]
```diff
@@ -0,0 +1,394 @@
...
+  }
+}
+
+Skills.propTypes = {
+  params: PropTypes.shape({
+    id: PropTypes.string,
+  }).isRequired,
+};
+
+export default Skills;
```
Comment (dane-johnson) [inline src/SkillUIDemo.jsx]: What is the purpose of this file?

**Review by dane-johnson — CHANGES_REQUESTED** [review]
I think you should remove the unused files from the directory.

File: src/AppDemo.jsx (hunk 1 above)
Comment (smokinGun) [reply src/AppDemo.jsx]: Said file has been removed @dane-johnson

File: src/ReviewPanel.jsx (hunk 2 above)
Comment (thebho) [reply src/ReviewPanel.jsx:10]: Can you clarify this @dane-johnson ?

Comment (dane-johnson) [reply src/ReviewPanel.jsx:10]: Like there used to be a little div here that I had the plus/minus in. I had thought that maybe sometime we would put more widgets there.

Comment (dane-johnson) [reply src/ReviewPanel.jsx:10]: Just an idea, not why I blocked the PR

**Review by dane-johnson — APPROVED** [review]

**Review by thebho — APPROVED** [review]

**Pull request merged**